css/_popup_menu.scss

```diff
@@ -30,7 +29,6 @@
         padding: 0 5px;
 
         &.disabled {
-            color: gray !important;
```

Comment:
Just removing some styles for the popup menu allows us to have the dark theme styles look better. We still need to use spans instead of li-s if we want this to look like a menu though. Left for later.